echo2_server.yaml

```diff
@@ -18,12 +18,12 @@ static_resources:
         address: 127.0.0.1
         port_value: 0
     filter_chains:
-      filters:
-        name: envoy.ratelimit
+    - filters:
+      - name: envoy.ratelimit
```

Comment:
Can we kill the ratelimit filter also? I think this is also going to cause issues for how the integration/extensions will be setup after all of my changes.

Reply:
Killed. 152055e